backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/location/impl/AddressBuilder.java

```diff
@@ -550,12 +532,9 @@ private void assertValidDisplaySequence(@NonNull final String displaySequence)
 		final boolean existsCON = isTokenFound(displaySequence, Addressvars.Contact.getName());
 		final boolean existsBPGReeting = isTokenFound(displaySequence, Addressvars.BPartnerGreeting.getName());
 
-		if ((existsBP && existsBPName) || (existsBP && existsBPGReeting)
-				|| (existsCON && existsBPName) || (existsCON && existsBPGReeting))
+		if ((existsBP && existsBPName) || (existsBP && existsBPGReeting))
 		{
-			throw new AdempiereException(MSG_AddressBuilder_WrongDisplaySequence)
-					.appendParametersToMessage()
-					.setParameter("displaySequence", displaySequence);
```

Comment:
is there any reason for removing this?

Reply:
I set them back. Did not realize that they were out. Not sure exactly how.